### DISCUSSION TITLE
Probing points: why 91/100 instead of 100/100?

### DISCUSSION THREAD

**Discussion by bulgaro80** [Jyers/Marlin#387] [Q&A] [answered]
Hi all,

I have checked within the old discussions without finding anything similar. 

Do you know why, running the Bed levelling with BLTouch, the printer stop at 91/100 instead of 100? I did not get any error message, just the standard message to confim and save the mesh...
![IMG_0023](https://user-images.githubusercontent.com/61660863/113578439-1d07bc00-9623-11eb-9b59-fa34009df67d.jpg)

What am I missing?

Many thanks in advance for any response.

Kind regards,
Stefano

**Comment by alphatrianguli** [comment]
Same on mine, 91/100. I guess, the mesh size might be smaller due to the BLTouch offset? But still works...

**Reply by darco415** [reply]
You need to run a  G29 P3 to fill in those points missed. G29 T to confirm...

**Reply by bulgaro80** [reply]
When I have to send those commands? After the bed leveling?

**Reply by darco415** [reply]
yes after BL....You would need a terminal access like Octoprint or pronterface....

**Reply by Jyers** [reply]
You don't actually need to run any G29 P3 commands. The firmware does it for you after creating a mesh.

**Comment by Jyers** [comment, marked as answer]
Some final clarification for anyone wondering. UBL creates a mesh that covers most of the bed, rather than just what the probe can reach. Therefore there are some points on the mesh which the probe is unable to reach. Those point are automatically inferred when the probing is completed in the leveling menu.

**Reply by alphatrianguli** [reply]
Thanks for the information.

Too bad, when finding the points for probing during a tilt, the printer / FW doesnt recognize, which areas of the bed are not accessible. As I stated in another discussion here on your Github, when trying to probe the very right end of the bed, the nut of the top right screw of the sled hits the gantry - or in other words: Due to the x offset of the BLTouch, there is no physical way of probing the very right edge of the bed. There would have to be a option to make the area of the tilting mesh smaller, to have an adequat margin of 2-3 cm around the bed edges.

To stop my sled from bumping into the gantry, I do not do a tilt at the moment and leave away the G29 J ;. Is there some G29 code, I can use, to choose the points to probe for tilt furthermore towards the center?

Thanks again for the clarification - much appreciated!

**Reply by Jyers** [reply]
I am aware of that issue and the margins are going to be adjusted in the next release. Sorry for the inconvenience at the moment.

**Reply by alphatrianguli** [reply]
You do not need to apologize at all :) You are doing a fantastic work with the firmware - keep it up! Looking forward to test the new version 👍
